src/src/PluginStructs/P131_data_struct.cpp

```diff
@@ -161,7 +163,7 @@ void P131_data_struct::cleanup() {
   delete gfxHelper;
   gfxHelper = nullptr;
 
-  delete matrix;
+  // delete matrix; // Doesn't have a virtual destructor
```

Comment:
If this cannot be deleted, how are we going to free the memory?
This is a memory leak.

You only need a virtual destructor if you have a pointer to the base class and calling the delete on the base class.

Reply:
I'll re-enable and just ignore the compiler warning 😺

Reply:
Or add a PR to the library for a virtual destructor ;)

Reply:
PR is created over there. I won't expect anything in a hurry, the last merged PR was ~20 months ago 🤔